apps/daffio/src/app/core/footer/sub-footer/sub-footer.component.ts

```diff
@@ -22,9 +28,15 @@ import { DaffContainerModule } from '@daffodil/design/container';
   standalone: true,
   imports: [
     FontAwesomeModule,
+<<<<<<< Updated upstream
+    DAFF_CALLOUT_COMPONENTS,
+    DAFF_CONTAINER_COMPONENTS,
+    DAFF_BASIC_BUTTON_COMPONENTS,
+=======
     DaffCalloutModule,
     DaffContainerModule,
-    DaffButtonModule,
+    DAFF_BUTTON_COMPONENTS,
```

Comment:
`DAFF_STROKED_BUTTON_COMPONENTS` should be used instead of `DAFF_BUTTON_COMPONENTS`.

``DAFF_BUTTON_COMPONENTS` imports all of the button types, so it's not necessary to use in this instance since only two types are used. And if `DAFF_BUTTON_COMPONENTS` is used, the other individual types (e.g. `DAFF_BASIC_BUTTON_COMPONENTS`) should not be used since this import will take care of all types.